Esri/arcgis-runtime-samples-ios
# [New] `Set max extent`

## Description

This PR implements `Set max extent` in `Maps` category.
Branch: https://github.com/Esri/arcgis-runtime-samples-ios/tree/Ting/New-SetMaxExtent-vNext/arcgis-ios-sdk-samples/Maps/Set%20max%20extent

## Linked Issue(s)

- `common-samples/issues/3477`

## How To Test

Run the sample, toggle the switch on to limit the map to a certain region. Toggle off to freely pan and zoom on the map.

Note it is expected that the extent is around half way extend to all 4 sides, though I don't know why - it seems to be how the maxExtent API designed.

## Screenshots

|On|Off|
|:-:|:-:|
|![set-max-extent](https://user-images.githubusercontent.com/9660181/166499270-36b86719-76ac-439e-a848-238e122749eb.png)|![Simulator Screen Shot - iPhone 11 - 2022-05-03 at 09 37 55](https://user-images.githubusercontent.com/9660181/166499381-87fe7695-bd04-454b-bcfd-d705d44543d9.png)|
